bin/start.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+const libPath = path.resolve(__dirname, '../lib/index.js');
+require('child_process').spawn(process.argv.shift(), [libPath], {
+  cwd: process.cwd(),
+  env: {
+    NODE_OPTIONS: '--experimental-repl-await',
```

Comment:
Let's enable top-level await only if `--experimental-repl-await` flag is provided to `baapan`. If it is not provided, baapan can behave in the usual manner.

With the flag:

```js
$ baapan --experimental-repl-await
> await Promise.resolve(1)
1
```

Without the flag:

```js
$ baapan
> await Promise.resolve(1)
Thrown:
await Promise.resolve(1)
^^^^^

SyntaxError: await is only valid in async function
```

Reply:
I think at we should keep a config file to enable disable these feature from now on. Like a file in user's  home directory.

Reply:
No. `baapan` should have the same behaviour as node repl itself where the configuration is provided by arguments.